CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [1.10.0](https://github.com/aws/jsii/compare/v1.9.0...v1.10.0) (2020-08-05)
+
+
+### Bug Fixes
+
+* **java:** user.xml repository ID was not sanitized ([#1812](https://github.com/aws/jsii/issues/1812)) ([4283af0](https://github.com/aws/jsii/commit/4283af0cb2338c31594c53b2ab040b7b183aec77))
+* **jsii:** document location of initializers in source ([#1806](https://github.com/aws/jsii/issues/1806)) ([3957827](https://github.com/aws/jsii/commit/3957827a5a730d51029313518caacb0e87c09b83))
```

Comment:
Nit: Should probably phrase as the issue that was fixed, instead of the solution.

Reply:
@nija-at I think the issue was them not being documented: https://github.com/aws/jsii/pull/1806

Got another suggestion?

Reply:
```suggestion
Location of initializers in source was not documented, even though it could have been
```

Reply:
```suggestion
* **jsii:** Location of initializers in source was not documented ([#1806](https://github.com/aws/jsii/issues/1806)) ([3957827](https://github.com/aws/jsii/commit/3957827a5a730d51029313518caacb0e87c09b83))
```